hyp3_sdk/hyp3.py

```diff
@@ -28,6 +29,7 @@ def __init__(self, api_url: str = HYP3_PROD, username: Optional = None, password
         """
         self.url = api_url
         self.session = get_authenticated_session(username, password)
+        self.session.headers.update({'User-Agent': f'{hyp3_sdk.__name__} v{hyp3_sdk.__version__}'})
```

Comment:
I think the convention is `tool + '/' + version without the v`

e.g. python requests looks like `python-requests/2.25.0`